pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
+	unversioned.TypeMeta
+	// Enabled must be true for the plugin to do anything.
+	// The plugin's actions can be disabled per-project with the project annotation
+	// quota.openshift.io/cluster-resource-override-enabled="false", so cluster admins
+	// can exempt infrastructure projects and such from the overrides.
+	Enabled bool
```

Comment:
Yeah in that case actually this admission controller will fail validation at initialization, because while you might deserve it, we can easily point out that you shot your foot.

I think I can get on board with no config == nil pointer. Specifically including a member Enabled always felt a bit redundant, and in my case was intended mainly to prevent having to check if all members are 0 (in two places). Nil pointer (or in JS, missing entry) does the job just as well.